docs/en/OLED_Display.md

```diff
@@ -0,0 +1,199 @@
+#### Placing it in separate folder may cause issues.
+
+```python
+oled.entries = [
+    ImageEntry(image="1.bmp", x=0, y=0),
+]
+keyboard.extensions.append(oled)
+```
+
+You can also make your images appear corresponding to specific layer
+
+```python
+oled.entries = [
+    ImageEntry(image="1.bmp", x=0, y=0, layer=0),
+    ImageEntry(image="2.bmp", x=0, y=0, layer=1),
+]
+keyboard.extensions.append(oled)
+```
+
+And/or side of your split keyboard
+
+```python
+oled.entries = [
+    ImageEntry(image="L1.bmp", x=0, y=0, side="L"),
+    ImageEntry(image="R1.bmp", x=0, y=0, side="R"),
+]
+keyboard.extensions.append(oled)
+```
+
+# Text
+To display longer texts you need to divide it to smaller phrases to fit width of your screen.
+
+You're able to freely positon of your text to place it wherever you want just by changing x and y values
+
+```python
+oled.entries = [
+    TextEntry(text="Layer = 1", x=0, y=0),
+    TextEntry(text="Macros", x=0, y=12),
+    TextEntry(text="Hey there!", x=0, y=24),
+]
+keyboard.extensions.append(oled)
+```
+### X and Y anchors
+It's helpfull with positioning of text.
+The values can be set "T" for Top, "M" for Middle and "B" for Bottom for X axis as well as "L" for Left, "M" for Middle and "R" for Right for Y axis.
+It sets the anchor point of given text and your text is moved and placed basing on this anchor point.
+For example for text in top right corner you need to set its anchor points Top Right and move text to far right position.
+For some more info about anchors check [Adafruit site](https://learn.adafruit.com/circuitpython-display-support-using-displayio/text). But keep in mind that KMK operates with `T`, `M`,`B` and `L`, `M`, `R` strings, not numbers.
+
+```python
+oled.entries = [
+    TextEntry(text="Layer = 1", x=128, y=0, x_anchor="R", y_anchor="T"), # text in Top Right corner
+    TextEntry(text="Macros", x=128, y=64, x_anchor="R", y_anchor="B"), # text in Bottom Right corner
+    TextEntry(text="Hey there!", x=64, y=32, x_anchor="M", y_anchor="M"), # text in the Middle of screen
+]
+keyboard.extensions.append(oled)
+```
+
+### Split
+As well as with images you can change displaying according to your layer or side of split keyboard.
+
+```python
+oled.entries = [
+    TextEntry(text="Longer text that", x=0, y=0, layer=0),
+    TextEntry(text="has been divided", x=0, y=12, layer=0, side="L"),
+    TextEntry(text="for an example", x=0, y=24, layer=0, side="R"),
+]
+keyboard.extensions.append(oled)
+```
+
+### Inverting
+inverts colours of your text. comes in handy for example as a good layer indicator.
+
+```python
+oled_ext = Oled(
+    entries=[
+        TextEntry(text='0 1 2 4', x=0, y=0),
+        TextEntry(text='0', x=0, y=0, inverted=True, layer=0),
+        TextEntry(text='1', x=12, y=0, inverted=True, layer=1),
+        TextEntry(text='2', x=24, y=0, inverted=True, layer=2),
+    ],
+)
+```
+
+# Fully working Example Code
+
```

Comment:
The reason not to include a "fully working example" is: it's "fully working" *only* in your specific case. Hardware setup and layout and so on doesn't add any usefull information here.
A "fully working example" is a "fully working keyboard" and belongs under `boards/`